website/project/metadata/hypothesis-testing-studies-using-youth-data.json

```diff
@@ -58,29 +58,34 @@
 		"example_text": "",
 		"required": false
 	}, {
-		"help_text": "",
+		"help_text": "Enter the datasets planned for this analysis",
 		"block_type": "question-label",
-		"display_text": "Enter the datasets planned for this analysis",
+		"display_text": "Datasets",
```

Comment:
Why did this change? Not seeing it in the spreadsheet.

Reply:
This is necessary because without the `Datasets` text there the red required asterik just floats in the middle of the page without anything next to it.